prime-angular-frontend/src/app/modules/enrolment/pages/obo-sites-page/obo-sites-page.component.html

```diff
@@ -32,61 +32,29 @@
               <ng-container *ngSwitchCase="CareSettingEnum.PRIVATE_COMMUNITY_HEALTH_PRACTICE"
                             formArrayName="communityHealthSites">
 
-                <ng-container *ngFor="let site of communityHealthSites.controls; let i = index; let last = last"
-                              [formGroupName]="i">
-
-                  <app-obo-site-form [form]="site"
-                                     [index]="i"
-                                     [careSettingCode]="careSetting.careSettingCode"
-                                     [site]="site"
-                                     [total]="communityHealthSites.controls.length"
-                                     [last]="last"
-                                     [jobNames]="jobNames"
-                                     [allowDefaultOption]="allowDefaultOption"
-                                     [defaultOptionLabel]="defaultOptionLabel"
-                                     (remove)="removeOboSite($event, careSetting.careSettingCode)"></app-obo-site-form>
+                <ng-container
+                              *ngTemplateOutlet="careSettingTemplate; context:{ $implicit: careSetting, careSettingSites: communityHealthSites };">
```

Comment:
To reduce line length you can use ngTemplateOutletContext to pass context, and use new lines to space out the passed object literal

Reply:
split into `[ngTemplateOutlet]` and `[ngTemplateOutletContext]`